client/src/components/Payment/Payment.tsx

```diff
@@ -1,4 +1,4 @@
-import { useState } from 'react';
+import { useState, useEffect } from 'react';
```

Comment:
Lets get rid of useEffect here if it's not being used

Reply:
will fix this in the morning, thanks for pointing that out